server/services/reportedAdjudicationsService.test.ts

```diff
@@ -149,7 +149,7 @@ describe('reportedAdjudicationsService', () => {
         {
           fromDate: moment('01/01/2021', 'DD/MM/YYYY'),
           toDate: moment('01/09/2021', 'DD/MM/YYYY'),
-          status: ReportedAdjudicationStatus.AWAITING_REVIEW,
+          status: [ReportedAdjudicationStatus.AWAITING_REVIEW],
```

Comment:
As it's just the one status you're passing in, it would have to be in an array, as the code that generates what's passed in as the status is 
`status: filter.status || allStatuses,` (in filterFromUIFilter function)

Reply:
same as previous comments